### PR TITLE
M64 release notes

### PR DESCRIPTION
**Pull request by infin8x**
M64 release notes blog post!

### CODE REVIEW

File: themes/default/content/blog/pulumi-release-notes-m64/index.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  - [`--json` flag for `pulumi {update|destroy|preview|refresh}`]({{<relref "/blog/pulumi-release-notes-m64#--json-flag-for-pulumi-updatedestroypreviewrefresh">}})
+  - [Python 3.6 support will be deprecated soon]({{<relref "/blog/pulumi-release-notes-m64#python-36-support-will-be-deprecated-soon">}})
+- Pulumi Service & Pulumi.com
+  - [Set the default organization for newly-created stacks]({{<relref "/blog/pulumi-release-notes-m64#new-resources-in-the-azure-native-provider">}})
+
+<!--more-->
+
+## Pulumi Registry, Pulumi Packages, & integrations
+
+### `dependsOn` can now await a fully-deployed Helm chart
```
Comment (joeduffy) [inline themes/default/content/blog/pulumi-release-notes-m64/index.md:33]: Will I be confused reading this if I'm a user of the more recent `Release` abstraction? For what it's worth, I'm not an avid user, but I am confused about how this behaves with the new resource type ... Maybe worth a mention?

**Comment by joeduffy** [comment]
Any community spotlight? (I'd love us to make this a standard thing for all release notes!)

File: themes/default/content/blog/pulumi-release-notes-m64/index.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+### `--json` flag for `pulumi {update|destroy|preview|refresh}`
+
+When automating or troubleshooting Pulumi operations, it can be helpful to see all of the details emitted by the Pulumi engine rather than the standard progress bar. Now, you can! Just add `--json` to your `pulumi {update|destroy|preview|refresh}` command.
+
+[Learn more about the new `--json` flag in this GitHub issue](https://github.com/pulumi/pulumi/issues/2390)
+
+### Python 3.6 support will be deprecated soon
+
+Python 3.6 will be end-of-lifed ("EOLed") on [December 23, 2021](https://www.python.org/downloads/#:~:text=2016-12-23-,2021-12-23,-PEP%20494). Around that time, Pulumi will drop support for Python 3.6. If you're using Python 3.6, consider upgrading to a newer version of Python. The Pulumi CLI will display a message when it runs a Pulumi program using Python 3.6.
```
Comment (joeduffy) [inline themes/default/content/blog/pulumi-release-notes-m64/index.md:222]: I wish we had a "standard EOL" policy we could link to here. And I would love if that EOL policy was not "right around the time the tech itself is EOL," but instead gives a quarter to two of grace period. Many enterprises have a tough time migrating and, unless we're saving ourselves a significant cost, not clear extra pain is worthwhile.

File: themes/default/content/blog/pulumi-release-notes-m64/index.md (hunk 1 above)
Comment (infin8x) [reply themes/default/content/blog/pulumi-release-notes-m64/index.md]: This is a great point, thanks. I've added a new note to clarify and also made subtle adjustments throughout to clarify that this is about the `Chart` resource, not the `Release` resource.

File: themes/default/content/blog/pulumi-release-notes-m64/index.md (hunk 2 above)
Comment (infin8x) [reply themes/default/content/blog/pulumi-release-notes-m64/index.md:226]: I've passed this feedback onto the team - @emiliza is working on an updated policy we can post for each language.

File: themes/default/content/blog/pulumi-release-notes-m64/index.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+## Pulumi CLI and core technologies
+
+In this milestone, we shipped Pulumi versions [3.16.1](https://github.com/pulumi/pulumi/releases/tag/v3.16.1) through [3.17.0](https://github.com/pulumi/pulumi/releases/tag/v3.17.0). The full list of changes in each version is available in the [changelog](https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md); read on to learn about some of the biggest changes.
+
+### Functions now support outputs
+
+When creating complex infrastructure, it's common to pass information about one resource to a function provided by another resource. Previously, doing this in Pulumi required you to use `apply` to get the information from the first resource and pass it into the function. Now, we've simplified the process so you can reference outputs directly from the function—no `apply` needed. To take advantage of these changes, make sure you've updated to the latest versions of Pulumi and the providers you're using.
```
Comment (t0yv0) [inline themes/default/content/blog/pulumi-release-notes-m64/index.md:214]: `function provided by another resource` --> perhaps just "to a function". It does not seem to me functions are associated with or provided by resources, perhaps only very indirectly (as in, some resources are always used with some functions together in practice).

File: themes/default/content/blog/pulumi-release-notes-m64/index.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
+---
+title: "Nov. 17 releases: `dependOn` Helm charts, new Elastic Cloud provider, functions support outputs, set the CLI's default organization"
+date: 2021-11-17T08:00:00-07:00
+draft: false
+allow_long_title: true
+meta_desc:
```
Comment (infin8x) [inline themes/default/content/blog/pulumi-release-notes-m64/index.md:6]:
```suggestion
meta_desc: The latest Pulumi updates include a new provider for Elastic Cloud, easier ways to create more complex infrastructure, better support for organizations, & more
```

**Pull request merged**